### ISSUE TITLE
Lightfall practice does not fire

### ISSUE THREAD

**Issue by MciReloaded** [Eclipse-Station/NEV-Northern-Light#1705]
The variant of the NT LR M "Lightfall" held within the shooting range does not fire. There is no battery within the device, and attempting to fire causes nothing to happen.

**Comment by Valanthe500** [comment]
Went testing this, and I'm unable to replicate the issue. 
- Neither the Aegis nor Civilian shooting ranges start with a practice gun
- Tested on the live server by spawning a practice laser (/obj/item/gun/energy/laser/practice) I was able to insert a medium cell and the weapon fired as normal.
- There is a small issue with the laser beam not appearing every time I fired, but that may just be due to latency or server lag. Uncertain at this time, but it is purely visual, testing with other laser weapons showed the same issue, but weapon performance is unaffected.
- I'll send a PR shortly that returns the practice lasers to both shooting ranges, and provides a simple medium cell for use. I tested once again locally to ensure the issue did not reappear.
